DESCRIPTION

```diff
@@ -18,6 +18,7 @@ License: MIT + file LICENSE
 URL: https://github.com/rstudio/bundle, https://rstudio.github.io/bundle/
 BugReports: https://github.com/rstudio/bundle/issues
 Imports:
+    fs,
```

Comment:
Could we bump fs and withr to Suggests? Look like we're only using these in this one bundler, so it'd be great if folks didn't have to install if they weren't using it.